tests/integration/test_pipelines.py

```diff
@@ -44,8 +46,10 @@ class PipelinesTestCase(IntegrationTestCase):
         await self.assertStatus(pipeline_iri, SwarmUI.Started)
         await self.assertNotExists(s=pipeline_iri, p=SwarmUI.restartRequested)
 
+
```

Comment:
Wrong number of spaces between function. How come the linter doesn't detect that...